client/lv_utils.py

```diff
@@ -268,7 +268,7 @@ def lv_remove(vg_name, lv_name):
     Remove a logical volume.
     """
     error.context("Removing volume /dev/%s/%s" %
-                  (vg_name, lv_name), logging.info)
+                  (vg_name, lv_name), logging.debug)
```

Comment:
Here I agree that the patchset should have started with `logging.debug` instead of this later commit. Let me finish the review to decide whether this is OK.